docs/2.8.0/docs/canton/usermanual/kms/external_key_storage.rst

```diff
@@ -0,0 +1,23 @@
+..
+   Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates.
+..
+   Proprietary code. All rights reserved.
+
+.. _external_key_storage:
+
+Configure External Key Storage and Usage with a Key Management Service (KMS)
```

Comment:
I think `Configure External Key Storage` is sufficient, WDYT? Seems implied that we're going to use them

```suggestion
Configure External Key Storage with a Key Management Service (KMS)
```